File: app/src/main/java/utils/TimeManager.java
package utils;

import org.joda.time.DateTime;
import org.joda.time.format.DateTimeFormat;
import org.joda.time.format.DateTimeFormatter;

public class TimeManager {
    DateTime startTime, endTime;
    DateTimeFormatter gameTimeFormat = DateTimeFormat.forPattern("ss");
    DateTimeFormatter startTimeFormat = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss");

    private static TimeManager instance = null;

    public TimeManager() {
    }

    public void setStartTime() {
        startTime = new DateTime();
    }

    public void setEndTime() {
        endTime = new DateTime();
    }

    public String getDate() {
        return startTimeFormat.print(startTime.getMillis());
    }

    public String gameTime() {
        return gameTimeFormat.print(
                endTime.minusHours(startTime.getHourOfDay())
                        .minusMinutes(startTime.getMinuteOfHour())
                        .minusSeconds(startTime.getSecondOfMinute())
        );
    }

    public static TimeManager getInstance() {
        if (instance == null) {
            instance = new TimeManager();
        }

        return instance;
    }
}

File: app/src/main/kotlin/utils/TimeManager.kt
package utils

import org.joda.time.DateTime
import org.joda.time.format.DateTimeFormat.forPattern
import org.joda.time.format.DateTimeFormatter

object TimeManager {
    lateinit var startTime: DateTime
    lateinit var endTime: DateTime

    val gameTimeFormat: DateTimeFormatter = forPattern("ss")
    val startTimeFormat: DateTimeFormatter = forPattern("yyyy-MM-dd HH:mm:ss")

    val date: String
        get() = startTimeFormat.print(startTime.millis)

    fun gameTime(): String = gameTimeFormat.print(
            endTime.minusHours(startTime.hourOfDay)
                    .minusMinutes(startTime.minuteOfHour)
                    .minusSeconds(startTime.secondOfMinute)
    )
}